core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-chips/dot-category-field-chips.component.html

```diff
@@ -5,7 +5,8 @@
         [removable]="true"
         [label]="category.value"
         tooltipPosition="top"
-        styleClass="p-chip-sm" />
+        styleClass="p-chip-sm"
+        (onRemove)="onRemove(category)" />
```

Comment:
use the output directly `remove.emit(category.key);`

Reply:
Fixed!